test/data/data.js

```diff
@@ -88,6 +97,30 @@ function populate(db) {
     }),
   );
 
+  prom.push(
+    insertDocument(db, {
+      $type: 'token',
+      $kind: 'entry',
+      $owner: 'b@b.com',
+      $id: 'myReadOnlyToken',
+      $creationDate: 0,
+      uuid: 'A',
+      rights: ['read'],
```

Comment:
to be clear, this token should only allow to "read" the entry with uuid "A". any other action on that entry and any action on other entries should not be allowed (unless the entry is public)